EXILED/Exiled.API/Features/Respawn.cs

```diff
@@ -272,7 +261,7 @@ public static void ModifyTickets(Faction team, int amount)
         /// </summary>
         /// <param name="faction"><see cref="SpawnableTeamType"/>'s faction.</param>
         /// <returns>Tickets of team or <c>-1</c> if team doesn't depend on tickets.</returns>
-        public static int GetTickets(SpawnableFaction faction)
+        public static int GetTokens(SpawnableFaction faction)
```

Comment:
Why introduce a break change here?

Reply:
Everything related to respawning will have to be reworked anyway so I thought we might as well name it correctly to not cause confusion

Reply:
Ummm, I think it's best to deprecate the old one and add a new one. It's not allowed nor needed to make break changes when unnecessary.

Reply:
actually this isn't a breaking change, this method was added here https://github.com/ExMod-Team/EXILED/commit/07113063bfa0ea99f2a2bbb909c3a5e39cdb8115#diff-446a27f4f248ca3b4c61f854e9c442c310dc859c4658c93a4d4145b897b631edR263-R269